src/target/linux.rs

```diff
@@ -96,9 +97,10 @@ impl LinuxTarget {
 
     /// Launches a new debuggee process
     pub fn launch(
-        path: &str,
+        path: &Path,
```

Comment:
We can use `impl AsRef<Path>` here instead (just as in the Rust's stdlib, e.g. in [`File::open`](https://doc.rust-lang.org/stable/std/fs/struct.File.html#method.open)) - then we can pass the path as a string.